docs/backup-restore.md

```diff
@@ -1,32 +1,101 @@
+
+To avoid mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, please follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+It is highly recommended to use the midtime of the pause as a restore point as we explained earlier.
+
+### DynamoDB
+
+You can restore the table to a point in time using the DynamoDB console or the AWS Command Line Interface (AWS CLI). The point-in-time recovery process restores to a new table.
+By default, the table can only be restored one by one.
+
+You can restore tables one by one from the [Amazon DynamoDB console](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html) using the following steps and use the midtime of the pause as a restore point
+ 
+* Restore with PITR of table A to another table B
+* Take a backup of the restored table B (assume the backup is named backup B)
+* Remove table A
+* Create a table named A with backup B
+
+Note that you need to follow the above steps because the application requires tables with the same name as before. In DynamoDB, table restoration is possible only with an alias, so you can restore the table with alias and delete the actual table and rename the restored table (alias) with an actual name.
+
+If you want to restore multiple tables with a single command, you can create a script to restore multiple tables using the AWS CLI commands.
 
-## Scalar DB server
+It is highly recommended to enable point-in-time recovery (PITR) for continuous backup creation and auto-scaling for better performance. Auto-scaling dynamically adjusts provisioned throughput capacity on your behalf in response to actual traffic patterns.
+DynamoDB will not enable PITR and auto-scaling by default after restoring.
 
-Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
-You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
-* Take backups of your databases during the paused duration with snapshot feature.
-* Restore to a point in the paused duration with point-in-time-restore (PITR) feature.
+You can enable continuous backup and auto-scaling using the schema loader or Amazon DynamoDB console. Configuring the continuous backup and auto-scaling is easier if you use the schema loader.
```

Comment:
I think this explanation sounds very confusing.
Isn't it better to say something like this?

It is recommended to re-apply schemas with the schema loader because it configures ...